_includes/releases/v21.2/v21.2.14.md

```diff
@@ -0,0 +1,87 @@
+## v21.2.14
+
+Release Date: August 1, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v21-2-14-security-updates">Security updates</h3>
+
+- Multi-region related built-in functions now have access control checks. [#83987][#83987]
+
+<h3 id="v21-2-14-sql-language-changes">SQL language changes</h3>
+
+- The error code reported when trying to use a system or virtual column in the [`STORING`](../v21.2/create-index.html#store-columns) clause of an index has been changed from `XXUUU (internal error)` to `0A000 (feature not supported)`. [#83649][#83649]
+- The error code reported when attempting to use a system column in an index as a key column has been changed from `XXUUU (internal error)` to `0A000 (feature not supported)`. [#83649][#83649]
+- [`AS OF SYSTEM TIME`](../v21.2/as-of-system-time.html) now takes the time zone into account when converting to UTC. For example: `2022-01-01 08:00:00-04:00` is treated the same as `2022-01-01 12:00:00` instead of `2022-01-01 08:00:00` [#84665][#84665]
+- Added additional logging for `COPY` statements to the [`SQL_EXEC` channel](../v21.2/logging.html#sql_exec) if the `sql.trace.log_statement_execute` cluster setting is set. [#84679][#84679]
+- An error message is now logged to the [`SQL_EXEC` channel](../v21.2/logging.html#sql_exec) when parsing fails. [#84679][#84679]
+
+<h3 id="v21-2-14-operational-changes">Operational changes</h3>
+
+- The application name that is part of a SQL session is no longer considered [`redactable`](../v21.2/configure-logs.html#redact-logs) information [#83558][#83558]
+- The [`cockroach debug zip`](../v21.2/cockroach-debug-zip.html) and [`cockroach debug merge-logs`](../v21.2/cockroach-debug-merge-logs.html) commands will now work with JSON formatted logs. [#83147][#83147]
+
+<h3 id="v21-2-14-db-console-changes">DB Console changes</h3>
+
+- Updated the message when there is no data on the selected time interval on [**Statements**](../v21.2/ui-statements-page.html) and [**Transactions**](../v21.2/ui-transactions-page.html) pages. [#84624][#84624]
```

Comment:
Nit: I would probably add an article in here after "on" 
so: "on the Statements and Transactions pages."